packages/checkbox/test/checkbox.test.ts

```diff
@@ -238,6 +238,17 @@ describe('Checkbox', () => {
         expect(el.checked).to.be.true;
     });
 
+    it('has name attribute', () => {
+        let el = testFixture.querySelector('#checkbox0') as Checkbox;
+
+        el = testFixture.querySelector('#checkbox1') as Checkbox;
+        expect(el.hasAttribute('name'));
+        expect(el.name).to.be.undefined;
+        expect(el.name == undefined);
```

Comment:
```suggestion
```

You can remove this line since it's the same as line 246.

Reply:
Remember what Hunter found about this syntax.

Reply:
> You can remove this line since it's the same as line 246.

Done!